crates/bevy_reflect/src/impls/std.rs

```diff
@@ -547,6 +545,217 @@ unsafe impl Reflect for Cow<'static, str> {
     }
 }
 
+impl<T: Reflect + Clone> GetTypeRegistration for Option<T> {
```

Comment:
Yeah that was on my todo list originally but I decided it could be done separately (since `Option<T>` appears to be the only case). If you wanna write a follow up that would be great!